main.rb

```diff
@@ -1,41 +1,5 @@
-#!/usr/bin/env ruby
-require_relative 'app'
-# rubocop:disable Metrics/CyclomaticComplexity
-def main
-  app = App.new
+require_relative 'starting'
 
-  response = nil
+def main; end
```

Comment:
- [ ] Currently, your main method does nothing. The code that starts the application is in the `starting.rb` file and it originates from the Start class. (proof is when I run `ruby starting.rb` in my terminal, the application starts). Based on the requirements, the entry point for the application should be in the `main.rb` file in a main method. Kindly implement this requirement. 👍